### PR TITLE
Final Pull Request

### PR DESCRIPTION
**Pull request by lfmnovaes**
## React Capstone Project
- Bootstrapped app from Create React App
- Added linters, babel, and workflow files
- Fetched data from [CoinGecko](https://www.coingecko.com/en/api/documentation) API
- Styled like the provided [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
- Added dynamic routing to multiple items
- Added Search input field to filter initial array
- Created unit tests for some components
- Created unit test for the redux code

### [Loom video](https://www.loom.com/share/347e920b8f7441e591fba84e15e2d6e1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,98 @@
...
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+</details>
+
+## Authors
```
Comment (rloterh) [inline README.md]: - [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely.

File: README.md [hunk 2]
```diff
@@ -1 +1,98 @@
...
+## Credits
+
+- [CoinGecko](https://www.coingecko.com/en/api/documentation) API
+- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) inspiration
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/lfmnovaes/rockets-missions/issues).
```
Comment (rloterh) [inline README.md]: - [ ] Kindly ensure that issue _link_ provided is pointing to the right project's issue.

File: README.md [hunk 3]
```diff
@@ -1 +1,98 @@
-# react-capstone
```
Comment (rloterh) [inline README.md:1]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon these aspects of your ReadMe. 
- [ ] Project description, how to set up, run and test this project should be detailed, clear, and well organized.
- [ ] The code aspects in each procedure should be in backticks`.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+} from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCryptos } from './redux/cryptos/cryptos';
+import Nav from './components/Nav';
+import CryptoList from './components/CryptoList';
+import CryptoDetails from './components/CryptoDetails';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+
+function App() {
```
Comment (rloterh) [inline src/App.js]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Luis, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request
- App is correctly deployed
- Good video presentation of the project
- Design matches provided template
- Redux is effectively used to manage state data
- Test cases are passing

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review:_
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (lfmnovaes) [reply README.md:1]: Even though the steps were already descriptive enough to run the application, I added one more instruction.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**